src/main/kotlin/io/github/inductiveautomation/kindling/statistics/categories/MetaStatistics.kt

```diff
@@ -0,0 +1,44 @@
+package io.github.inductiveautomation.kindling.statistics.categories
+
+import io.github.inductiveautomation.kindling.statistics.GatewayBackup
+import io.github.inductiveautomation.kindling.statistics.Statistic
+import io.github.inductiveautomation.kindling.statistics.StatisticCalculator
+import io.github.inductiveautomation.kindling.utils.asScalarMap
+import io.github.inductiveautomation.kindling.utils.executeQuery
+
+data class MetaStatistics(
+    val uuid: String?,
+    val gatewayName: String,
+    val edition: String,
+    val role: String,
+    val version: String,
+    val initMemory: Int,
+    val maxMemory: Int,
+) : Statistic {
+    @Suppress("SqlResolve")
+    companion object : StatisticCalculator<MetaStatistics> {
+        private val SYS_PROPS =
+            """
+            SELECT *
+            FROM
+                sysprops
+            """.trimIndent()
+
+        override suspend fun calculate(backup: GatewayBackup): MetaStatistics {
+            val sysPropsMap = backup.configDb.executeQuery(SYS_PROPS).asScalarMap()
+
+            val edition = backup.info.getElementsByTagName("edition").item(0)?.textContent
+            val version = backup.info.getElementsByTagName("version").item(0).textContent
+
+            return MetaStatistics(
+                uuid = sysPropsMap["SYSTEMUID"] as String?,
+                gatewayName = sysPropsMap.getValue("SYSTEMNAME") as String,
+                edition = edition.takeUnless { it.isNullOrEmpty() } ?: "Standard",
+                role = backup.redundancyInfo.getProperty("redundancy.noderole"),
+                version = version,
+                initMemory = backup.ignitionConf.getProperty("wrapper.java.initmemory").toInt(),
```

Comment:
These need to be `.takeWhile { it.isDigit() }.toInt()` in the case where there is a comment after the parameter. (Yeah this has happened)